packages/block-library/src/list-item/index.js

```diff
@@ -18,4 +18,14 @@ export const settings = {
 	icon,
 	edit,
 	save,
+	merge( attributes, attributesToMerge ) {
+		const { content } = attributesToMerge;
+		if ( ! content || content === '<li></li>' ) {
```

Comment:
Checking for `content === '<li></li>'` is no longer needed because there are just the list item contents?

Reply:
The merge function should be a simple as the paragraphs:

```js
merge( attributes, attributesToMerge ) {
		return {
			content:
				( attributes.content || '' ) +
				( attributesToMerge.content || '' ),
		};
	},
```

Reply:
Although it does make sense to also spread the `...attributes` :)

Reply:
Hm.. yes, and also doesn't seem to check at all because at the time of execution with have inserted: `const START_OF_SELECTED_AREA = '\u0086';` so it's never empty.

Reply:
GH hadn't refreshed the page 😄